pages/events/seminar-series.vue

```diff
@@ -1,19 +1,57 @@
 <template>
   <main class="event-page seminar-series-page">
-    <SeminarSeriesHeader class="seminar-series-page__header" :next-event="nextEvent" :past-events="pastEvents" />
```

Comment:
The `SeminarSeriesEvent.vue` component can be safely deleted now since it's now longer in use. 🙂